dsgrid/cli/registry.py

```diff
@@ -820,7 +1004,17 @@ def replace_dataset_dimension_requirements(
         return 1
 
 
-@click.command(name="list-dimension-query-names")
+_list_project_dimension_query_names_epilog = """
+Examples:\n
+$ dsgrid registry projects list-dimension-query-names\n
+$ dsgrid registry projects list-dimension-query-names --exclude-subset\n
+$ dsgrid registry projects list-dimension-query-names --exclude-supplemental\n
+"""
```

Comment:
Can you filter by project_id?

Reply:
project_id is a required argument. I apparently forgot to include it.